### DISCUSSION TITLE
Lost the miner's owner, how to change the owner of miner

### DISCUSSION THREAD

**Discussion by yangjian102621** [filecoin-project/lotus#3955] [General Q&A] [answered]
@magik6k @whyrusleeping My lotus daemon node crashed, and i did not backup the wallet key. 
Is any way to update the miner's owner(or change the worker address) with a new address?  I want to keep my miner running. 
thansk.

**Comment by jennijuju** [comment, marked as answer]
Unfortunately, no. Always back up your keys! To be able to update the owner address of a miner with the existing owner key is requested in #3951 .

Also, moving this to discussion since it’s more a question than an issue.

**Reply by jennijuju** [reply]
You will also need the owner key for adding a control address
